yt/tests/test_load_sample.py

```diff
@@ -196,3 +196,7 @@ def test_data_dir_broken():
     )
     with pytest.warns(UserWarning, match=msg):
         load_sample("ToroShockTube")
+
+
+def test_filename_none():
+    assert load_sample() is None
```

Comment:
is there a `with pytest. ???` variation that i can use here to check the message that gets printed?